song/song_server.py

```diff
@@ -130,6 +131,12 @@ def reset_handler(self, _, content):
         self.osculator_client.send_message(settings.SONG_MIDICC_ADDRESS + '{}'.format(self.tonality.ccnr),
                                            self.tonality.synth.reset_values[str(self.tonality.ccnr)])
 
+    def article_handler(self, _, content):
+        print('article? ', content)
+        data = json.dumps({'newLine' : content})
+        self.audience_client.send_message(settings.DISPLAY_ARTICLE_ADDRESS, data)
+
```

Comment:
Why not send the article directly from `performer` to `audience`, without passing `song`?